### DISCUSSION TITLE
Background noise on spatial slides

### DISCUSSION THREAD

**Discussion by torsteintengs** [satijalab/seurat#7364] [Q&A]
Hi!

I have done a few slides using the Visium Spatial transcriptomics platform from 10x genomics, and I have seen some genes that seem to have areas of expression where I suspect they should not. To check how much the mRNA has leaked from one spot to the next, I would like to see the expression pattern for specific genes for the entire slide/all the spots (not just the regions covered by tissue) to get an idea of how much the mRNA has moved to places where it does not belong during sample preparation. Any idea how to do this?

**Comment by limin321** [comment]
> Hi!
> 
> I have done a few slides using the Visium Spatial transcriptomics platform from 10x genomics, and I have seen some genes that seem to have areas of expression where I suspect they should not. To check how much the mRNA has leaked from one spot to the next, I would like to see the expression pattern for specific genes for the entire slide/all the spots (not just the regions covered by tissue) to get an idea of how much the mRNA has moved to places where it does not belong during sample preparation. Any idea how to do this?

We did some analysis on our manuscript:
https://www.biorxiv.org/content/10.1101/2022.12.05.519162v1

**Reply by torsteintengs** [reply]
Thanks! I also realized that you can do this if you use manual fiducial alignment (https://support.10xgenomics.com/spatial-gene-expression/software/visualization/latest/alignment). In some of the samples, we had a lot of mRNA leakage. Probably due to non-optimal tissue permeabilization.
